### PR TITLE
Effects rewrite

### PR DESCRIPTION
**Pull request by jzaefferer**
Addresses some more of #242.

### CODE REVIEW

File: entries/cssClip.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<entry name="cssClip" type="method" return="jQuery">
+	<title>.cssClip()</title>
+	<desc>getter/setter for an object version of the CSS <code>clip</code> property</desc>
```
Comment (scottgonzalez) [inline entries/cssClip.xml]: Start with a capital letter.

Comment (scottgonzalez) [inline entries/cssClip.xml]: End with a period.

File: entries/cssClip.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	<added>1.8</added>
```
Comment (scottgonzalez) [inline entries/cssClip.xml]: 1.12

File: entries/cssClip.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	<signature>
+		<argument name="dimensions" type="Object" optional="true">
+			<desc>When setting the CSS <code>clip</code> property, specify the <code>top</code>, <code>right</code>, <code>bottom</code> and <code>left</code>, properties to use the the <code>rect()</code> style.</desc>
```
Comment (scottgonzalez) [inline entries/cssClip.xml]: "the the"

File: entries/cssClip.xml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+	<example>
+		<height>175</height>
+		<css><![CDATA[
+	.clipped {
+		position: absolute;
+		width: 150px;
+		height: 150px;
+		background: #3b679e;
+		background: linear-gradient(to bottom, #3b679e 0%,#2b88d9 50%,#207cca 51%,#7db9e8 100%);
+	}
```
Comment (scottgonzalez) [inline entries/cssClip.xml]: Spaces after commas.

File: entries/cssClip.xml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+]]></code>
+		<html><![CDATA[
+Click anywhere!
+<div class="clipped"></div>
+]]></html>
+	</example>
+	<category slug="methods"/>
+	<category slug="overrides"/>
+	<category slug="utilities"/>
```
Comment (scottgonzalez) [inline entries/cssClip.xml]: Seems like effects would be reasonable too.

File: entries/cssClip.xml (hunk 3 above)
Comment (jzaefferer) [inline entries/cssClip.xml]: I need a prose lint that highlights "the the" as an error.

**Comment by jzaefferer** [comment]
Addressed everything. Added to both "effects" and "effects-core" categories.

**Comment by scottgonzalez** [comment]
Looks good.

**Pull request merged**